certified-connectors/LMS365/apiDefinition.swagger.json

```diff
@@ -21,6 +23,9 @@
   "paths": {
     "/odata/v2/Enrollments({Id})/Approve": {
       "post": {
+        "consumes": [
+          "application/json"
+        ],
         "responses": {
           "default": {
             "description": "default",
```

Comment:
@elearningforce-ag Default responses - I would recommend you to replace ALL the "default" responses with expected status codes, otherwise users may see errors about incorrect schema instead of real error responses from your service in OpenAPI mode.